Core.DomainModel/Return.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Net;
+
+namespace Core.DomainModel
+{
+    public enum ReturnType
+    {
+        Ok,
+
+        NotFound,
+
+        Forbidden,
+
+        Error
+    }
+
+    public class Return<T>
```

Comment:
- rename Return --> Result
- Tilføj en status parameter: Return<TResult,TStatus> så man ikke er bundet til den der return type.
- ReturnType skal konkretiseres f.eks. OperationResult
- Lad være med at bruge public fields. Du skal lave read-only properties og lave en private constructor der tager begge parametre. For at lave instanser, skal du have to factory metoder Result.Ok(value) som kalder private ctor med (OK,value) og en Fail(code) der kalder base ctor med (code,null)